crates/snapbox/src/data.rs

```diff
@@ -20,6 +22,14 @@ impl Data {
         }
     }
 
+    /// Mark the data as json (no post-processing)
+    #[cfg(feature = "json")]
+    pub fn json(raw: impl Into<Vec<serde_json::Value>>) -> Self {
```

Comment:
Isn't `Vec<Value>` fairly prescriptive on what data we can compare?  If you are trying to support [ndjson](http://ndjson.org/), then that is a separate format from json

Reply:
My concern isn't with `Into` its with `Vec`

Reply:
Do you think it should be just `Value`?